<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetLogin</class>
 <widget class="QDialog" name="SetLogin">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Set Login</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username:</string>
      </property>
      <property name="buddy">
       <cstring>usernameBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameBox">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="buddy">
       <cstring>passwordBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passwordBox">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="confirmPasswordBox">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="confirmPasswordLabel">
      <property name="text">
       <string>Password Again:</string>
      </property>
      <property name="buddy">
       <cstring>confirmPasswordBox</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
